tests/test_ray.py

```diff
@@ -319,6 +319,41 @@ def test_circularGrid():
         np.testing.assert_allclose(ray.v[1]*1.2, ycos)
 
 
+@timer
+def test_uniformCircularGrid():
+    dist = 10.0
+    outer = 4.1
+    inner = 0.5
+    xcos = 0
+    ycos = 0
+    zcos = -1
+    nray = 100000
+    wavelength = 500e-9
+    flux = 1.
+    medium = batoid.ConstMedium(1.)
+    seed = 0
+
+    rays = batoid.uniformCircularGrid(dist, outer, inner, xcos, ycos, zcos, nray, wavelength, flux,
+                                      medium, seed=seed)
+    radius = np.sqrt(np.sum(rays.r[:,:2] * rays.r[:,:2], axis=1))
+    angle = np.arctan2(rays.r[:,1], rays.r[:,0])
+
+    np.testing.assert_almost_equal(radius.max(), outer, decimal=1)
+    np.testing.assert_almost_equal(radius.min(), inner, decimal=1)
+
+    np.testing.assert_almost_equal(angle.mean(), 0, decimal=2)
+
+    # test seed, reproducibility
+    rays2 = batoid.uniformCircularGrid(dist, outer, inner, xcos, ycos, zcos, nray, wavelength, flux,
+                                       medium, seed=seed)
+    newseed = 666
+    rays3 = batoid.uniformCircularGrid(dist, outer, inner, xcos, ycos, zcos, nray, wavelength, flux,
+                                       medium, seed=newseed)
+
+    np.testing.assert_allclose(rays.r, rays2.r)
```

Comment:
You can just use `assert rays == rays2` here (they should be exactly equal) and `assert rays2 != rays3` below.